openedx_webhooks/github_views.py

```diff
@@ -46,29 +49,34 @@ def pull_request():
         return "PONG"
 
     pr = event["pull_request"]
+    PR_NUMBER = pr['number']
```

Comment:
Agreed, especially since there's syntax for explicit declaration of `const` vs. `let`.